### PR TITLE
chore: angular 17

### PR DESCRIPTION
**Pull request by sevaru**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the new behaviour?

Use angular 17 in example apps

### CODE REVIEW

File: apps/demo/src/app/modules/angular-installation/angular-installation.template.html [hunk 1]
```diff
@@ -8,14 +8,14 @@ <h2>Package installation</h2>
 
     <p>Install the core package if you do not have it yet:</p>
 
-    <code>npm i @microzord/core</code>
+    <code>npm i &#64;microzord/core</code>
```
Comment (splincode) [inline apps/demo/src/app/modules/angular-installation/angular-installation.template.html:11]: why?

**Review by splincode — DISMISSED** [review]

File: apps/demo/src/app/modules/angular-installation/angular-installation.template.html (hunk 1 above)
Comment (sevaru) [reply apps/demo/src/app/modules/angular-installation/angular-installation.template.html:11]: automigration from nx, i'll take a look

Comment (sevaru) [reply apps/demo/src/app/modules/angular-installation/angular-installation.template.html:11]: error NG5002: Incomplete block "microzord". If you meant to write the @ character, you should use the "@" HTML entity instead.

Comment (sevaru) [reply apps/demo/src/app/modules/angular-installation/angular-installation.template.html:11]: https://github.com/angular/angular/issues/52366

Comment (splincode) [reply apps/demo/src/app/modules/angular-installation/angular-installation.template.html:11]: Oh, sure
image

**Pull request merged**